src/signals/incident-management/containers/Dashboard/components/ModuleTitle.tsx

```diff
@@ -12,7 +12,7 @@ export const ModuleTitle = ({ title, subtitle, amount }: Props) => (
   <>
     <Title>
       {title}
-      {amount && <Amount> 422</Amount>}
+      {amount && <Amount>{amount}</Amount>}
```

Comment:
You should add the right amount of space on the left of the number here. Else, everyone that uses this component needs to add the spaces in the props, which is not what we want. 
Also, I guess it should be one space and the design is wrong, but check with Linda.

Reply:
Did it. Linda said two spaces.